content/docs/hooks-reference.md

```diff
@@ -452,7 +452,7 @@ function FancyInput(props, ref) {
 FancyInput = forwardRef(FancyInput);
 ```
 
-In this example, a parent component that renders `<FancyInput ref={inputRef} />` would be able to call `inputRef.current.focus()`.
+In this example, a parent component that renders `<FancyInput ref={fancyRef} />` can now call `fancyRef.focus()` to focus the underlying `<input />`.
```

Comment:
It's unclear when this "now" is. What does this new sentence clarify for you?

Reply:
Oh sorry, yes I misunderstood.